packages/grafana-ui/src/components/Monaco/CodeEditor.tsx

```diff
@@ -12,6 +12,7 @@ export interface CodeEditorProps {
 
   readOnly?: boolean;
   showMiniMap?: boolean;
+  hideLineNumbers?: boolean;
```

Comment:
Add this as a knob in the story too :+1:

Reply:
wonder if we should invert it, so that it is showLineNumbers instead? 
Feels more naturual to hide something by setting it to false rather than true

Reply:
except that the "normal" case is to show them -- but I could go either way